### PR TITLE
feat: small enhancement

### PR DESCRIPTION
**Pull request by peterdeme**

### CODE REVIEW

File: src/stream-net/IBatchOperations.cs [hunk 1]
```diff
@@ -9,7 +9,8 @@ public interface IBatchOperations
         Task AddToMany(Activity activity, IEnumerable<string> feedIds);
         Task FollowMany(IEnumerable<Follow> follows, int activityCopyLimit = 300);
         Task<IEnumerable<Activity>> GetActivities(IEnumerable<string> ids = null, IEnumerable<ForeignIDTime> foreignIDTimes = null);
-        Task<IEnumerable<EnrichedActivity>> GetEnrichedFlatActivities(IEnumerable<string> ids = null, ReactionOption reactions = null, IEnumerable<ForeignIDTime> foreignIDTimes = null);
+        Task<IEnumerable<EnrichedActivity>> GetEnrichedFlatActivities(IEnumerable<string> ids, GetOptions options = null);
```
Comment (ferhatelmas) [inline src/stream-net/IBatchOperations.cs:12]: this is for activities, not tied to feed types so I think it's better to drop flat keyword

Comment (peterdeme) [reply src/stream-net/IBatchOperations.cs]: ✅

File: src/stream-net/BatchOperations.cs [hunk 2]
```diff
@@ -135,30 +135,35 @@ public async Task<IEnumerable<Activity>> GetActivities(IEnumerable<string> ids =
...
-                request.AddQueryParameter("ids", string.Join(",", ids));
-            }
 
-            if (foreignIDTimes != null)
-            {
-                request.AddQueryParameter("foreign_ids", string.Join(",", foreignIDTimes.Select(f => f.ForeignID)));
-                request.AddQueryParameter("timestamps", string.Join(",", foreignIDTimes.Select(f =>
-                        f.Time.ToString("s", System.Globalization.CultureInfo.InvariantCulture))));
-            }
+            request?.AddQueryParameter("ids", string.Join(",", ids));
```
Comment (ferhatelmas) [inline src/stream-net/BatchOperations.cs:160]: this needs conditional add

Comment (peterdeme) [reply src/stream-net/BatchOperations.cs]: ✅

**Review by ferhatelmas — APPROVED** [review]

**Comment by peterdeme** [comment]
@ferhatelmas sure. I've been doing amended commits for 4 years and I'm so used to it haha. But I'll try!

**Comment by ferhatelmas** [comment]
>  I'll try!

Thanks! My concern was related to conventional commits if our pipeline can work with multiple commits or not

**Pull request merged**